django_declarative_apis/machinery/attributes.py

```diff
@@ -392,6 +400,9 @@ def _resolve_maybe_callable(self, owner_instance, maybe_callable):
             return maybe_callable
 
     def _run_task(self, owner_instance):
+        if self.execute_unless and self.execute_unless(owner_instance):
```

Comment:
This could still result in a type error (`execute_unless` has been given a non-callable value after the fact). Think that case should be handled?

Reply:
We would handle it by catching the error and raising a new error, which doesn't seem fundamentally different from just letting the original error propagate.  The developer would have to be up to some pretty serious shenanigans in order to trigger this case; I don't mind punishing them with the underlying TypeError in this case :-)

Reply:
Fair enough.